src/components/footer/index.md.njk

```diff
@@ -45,6 +45,19 @@ You can add links to:
 
 Use ‘Privacy’, ‘Accessibility’, ‘Cookies’ and ‘Terms and conditions’ for the link text.
 
+## Use localisation to translate ‘footer’ into other languages
+
+Read how to [set up localisation with Nunjucks and Javascript](/get-started/localisation/).
```

Comment:
Change this sentence to remove 'set up'. Maybe to 'Read how to use localisation with...'